src/blocks/blocks/accordion/group/edit.js

```diff
@@ -45,26 +59,125 @@ const Edit = ({
 		return () => unsubscribe( attributes.id );
 	}, [ attributes.id ]);
 
+	// Make it true for old users if they have more than one accordion item
+	// with initiallyOpen === true, and false by default otherwise
+	if ( attributes.alwaysOpen === undefined ) {
+		const children = select( 'core/block-editor' ).getBlocksByClientId( clientId )[0].innerBlocks;
+		setAttributes({ alwaysOpen: 1 < children.filter( child => true === child.attributes.initialOpen ).length });
+	}
+
 	const getValue = field => getDefaultValueByField({ name, field, defaultAttributes, attributes });
 
+	const getBorderValue = ( type, position, property ) => {
+		const border = 'header' === type ?
+			getValue( 'headerBorder' ) :
+			( 'content' === type ?
+				getValue( 'contentBorder' ) :
+				undefined );
+
+		if ( ! border ) {
+			return '';
+		}
+
+		// for backward compatibility purposes
+		if ( 'color' === property ) {
+			return border[position] ?
+				border[position].color :
+				border.color || getValue( 'borderColor' );
+		}
+
+		return border[position] && border[position][property] ? border[position][property] : border[property];
+	};
+
+	const boxShadow = getValue( 'boxShadow' );
+
 	const inlineStyles = {
 		'--titleColor': getValue( 'titleColor' ),
 		'--titleBackground': getValue( 'titleBackground' ),
-		'--borderColor': getValue( 'borderColor' ),
-		'--contentBackground': getValue( 'contentBackground' )
+		'--contentBackground': getValue( 'contentBackground' ),
+		'--fontFamily': getValue( 'fontFamily' ),
+		'--fontVariant': getValue( 'fontVariant' ),
+		'--fontStyle': getValue( 'fontStyle' ),
+		'--textTransform': getValue( 'textTransform' ),
+		'--letterSpacing': getValue( 'letterSpacing' ) ? getValue( 'letterSpacing' ) + 'px' : undefined,
+		'--fontSize': getValue( 'fontSize' ) ? getValue( 'fontSize' ) + 'px' : undefined,
+		'--boxShadow': boxShadow.active && `${boxShadow.horizontal}px ${boxShadow.vertical}px ${boxShadow.blur}px ${boxShadow.spread}px ${hex2rgba( boxShadow.color, boxShadow.colorOpacity )}`
 	};
 
+	const addBorderStyle = ( type ) => {
+		[ 'Top', 'Right', 'Bottom', 'Left' ].forEach( position => {
+			[ 'Width', 'Style', 'Color' ].forEach( prop => {
+				if ( 'content' === type && 'Top' === position ) {
```

Comment:
I see that you made the words to start with uppercase and using `.toLowerCase()` to have them in lowercase. You could write them directly in lowercase.